### PR TITLE
Bug Fix: wrong invokation to call ./scripts/install_all.bat which let dependency check always pass on Windows platform

### PR DESCRIPTION
**Pull request by haroldrandom**
* Bug fix: issue #9853 to make dependency check valid on Windows platform

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by haroldrandom** [comment]
FYI.

As you can see, in our previous dependency_check job **Verify src/azure-cli/requirements.*.Windows.txt**,

build [20190702.02](https://dev.azure.com/azure-sdk/public/_build/results?buildId=60891&view=logs&j=796343fd-f04d-59ce-7e73-c4eab21e4249&t=3e5b4737-4f47-5027-dc25-be482d8c1eaf&l=342) exited and return code 0 even though script detectes error. Which is not reasonable. 

build [20191031.26](https://dev.azure.com/azure-sdk/public/_build/results?buildId=176944&view=logs&j=796343fd-f04d-59ce-7e73-c4eab21e4249&t=3e5b4737-4f47-5027-dc25-be482d8c1eaf&l=390) exited and return code 0 even though script interupted.

So, whatever situation, dependency check of CI will always green even though it detects error on Windows platform.

My changes of [my first commit](https://github.com/Azure/azure-cli/pull/11094/commits/5faa98f1dd3e217cbe855792963672d35d50bc4f) can reproduce those build's situation.

File: scripts/install_full.bat [hunk 1]
```diff
@@ -11,5 +11,4 @@ set cmd="python .\scripts\get-python-version.py"
 for /f "tokens=*" %%v in (' %cmd% ') do set SHORT_PY_VER=%%v
 pip install -r .\src\azure-cli\requirements.%SHORT_PY_VER%.Windows.txt
 
-echo %OUTPUT%
```
Comment (fengzhou-msft) [inline scripts/install_full.bat]: what is the value of %OUTPUT%? Why remove it?

Comment (haroldrandom) [reply scripts/install_full.bat]: %OUTPUT% is not a variable in those *.bat scripts context,  Azure DevOps and Windows Batch sctipt. The functionality is going to ECHO a non-existing variable. Can tell from [line 360 of build 20190702.7](https://dev.azure.com/azure-sdk/public/_build/results?buildId=60891&view=logs&j=796343fd-f04d-59ce-7e73-c4eab21e4249&t=3e5b4737-4f47-5027-dc25-be482d8c1eaf&l=360) . 

So, I think it's trivial.

File: scripts/ci/dependency_check.bat [hunk 2]
```diff
@@ -9,8 +9,7 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 del baseline_deps.txt
 
 Rem Install everything from our repository first.
-.\scripts\install_full.bat
-if %errorlevel% neq 0 exit /b %errorlevel%
+call .\scripts\install_full.bat
```
Comment (fengzhou-msft) [inline scripts/ci/dependency_check.bat]: what happens with `call` if install_full.bat returns non-zero exit code?

Comment (haroldrandom) [reply scripts/ci/dependency_check.bat]: the script won't fail as all it is going to do, is `pip install`

Comment (fengzhou-msft) [reply scripts/ci/dependency_check.bat]: should we still keep `if %errorlevel% neq 0 exit /b %errorlevel%`, just in case?

Comment (haroldrandom) [reply scripts/ci/dependency_check.bat]: I could do so, but it's not necessary I think

Comment (haroldrandom) [reply scripts/ci/dependency_check.bat]: Adding it back for further modification of ./scripts/install_all.bat

**Comment by MyronFanQiu** [comment]
talked with @haroldrandom offline. LGTM.

**Pull request merged**